gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/ExplodeBundle.java

```diff
@@ -37,6 +38,14 @@
         this.entityLinkerRegistry = entityLinkerRegistry;
     }
 
+    @SuppressWarnings("squid:S1075")
+    boolean bundleContainsFolderPath(Bundle bundle, String folderPath) {
```

Comment:
Can be package private here, also double check if we still need to suppress the warning

Reply:
I believe by default, with no modifier it's package private.